browser/model/jbds-autoinstall.js

```diff
@@ -12,7 +12,7 @@ class JbdsAutoInstallGenerator {
   }
 
   generate(jbdsInstallDir, jdkInstallDir) {
-    let jbdsVersion = "9.1";
+    let pjson = require('../../package.json');

```

Comment:
Could DevStudio product build just generate contents.json (similar to eclipse p2 repo) with some additional info
{
   "name": "Red Hat Developer Studio Nightly Build",
   "version": "10.0.0.Alpha2",
   "timestamp": "<buildQualifier>",
   "installer": "filename"
}

Installer could use base URL for contents.json and calculate url to download installer file and sha256 file for verification.